src/System.Private.ServiceModel/src/System/ServiceModel/Channels/TransportDefaults.cs

```diff
@@ -140,8 +140,8 @@ public static class BinaryEncoderDefaults
 
     public static class TransportDefaults
     {
-        public const bool ExtractGroupsForWindowsAccounts = true;
         //public const bool ExtractGroupsForWindowsAccounts = SspiSecurityTokenProvider.DefaultExtractWindowsGroupClaims;
```

Comment:
If you are changing the default value, change the value of SspiSecurityTokenProvider.DefaultExtractWindowsGroupClaims and add a comment there that the default was different on Desktop.